Makefile

```diff
@@ -46,11 +46,11 @@ build-docs:
 	$(MAKE) -C docs html
 	$(MAKE) -C docs doctest
 
-validate-docs:
+validate-docs: build-docs
```

Comment:
nit: Curious on the deviation from the other projects here. Leaving `validate-docs` as just validating, `build-docs` as just building and `docs` as the combo of both (`build-docs` followed by `validate-docs` and opening) seems to be standard across the projects.

Reply:
Yeah, good question. I think there's value in running `validate docs` on the lint command instead of just `make build-docs`, but running `make docs` via tox was giving me an error that it didn't know what `open` was. I'll add a separate command for tox to run that builds and validates but doesn't open. I'll add to the template as well.